src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -428,12 +439,11 @@ internal static bool IsEdgeSupportedType(Type type)
                 type = type.GetElementType();
             }
 
-            if (type == typeof(int) ||
-                type == typeof(double) ||
+            if (type.IsPrimitive ||
```

Comment:
This is covering a much wider range of types. Any concerns with breaking changes due to the potential change in behavior? (e.g. `DateTime` will no longer be passed as string, correct?)